src/main/java/net/sf/jabref/gui/dbproperties/FieldFormatterCleanupsPanel.java

```diff
@@ -44,7 +44,7 @@
 
     private JButton resetButton;
 
-    private FieldFormatterCleanups defaultFormatters;
+    private final FieldFormatterCleanups defaultFormatters;
```

Comment:
Sure, it was only line 233 which I personally changed, Eclipse took care of the rest.